internal/controllers/topology/cluster/patches/engine_test.go

```diff
@@ -904,6 +1095,28 @@ func setupTestObjects() (*scope.ClusterBlueprint, *scope.ClusterState) {
 							Replicas: pointer.Int32(5),
 						},
 					},
+					MachinePools: []clusterv1.MachinePoolTopology{
+						{
+							Metadata: clusterv1.ObjectMeta{},
+							Class:    "default-mp-worker",
+							Name:     "default-mp-worker-topo1",
+							Variables: &clusterv1.MachinePoolVariables{
+								Overrides: []clusterv1.ClusterVariable{
+									{
+										Name:           "default-mp-worker-infra",
+										DefinitionFrom: "inline",
+										Value:          apiextensionsv1.JSON{Raw: []byte(`"value1"`)},
```

Comment:
Let's use a different value than the one for machinedeployments here, so we ensure we don't use the one from MD by accident


```suggestion
										Value:          apiextensionsv1.JSON{Raw: []byte(`"value2"`)},
```